### ISSUE TITLE
training objective

### ISSUE THREAD

**Issue by lwzhaojun** [BoyuanJiang/matching-networks-pytorch#8]
The training objective in the original paper is theta, should the loss value be theta? Why is loss in your program a cross entropy loss?
